### DISCUSSION TITLE
Recognizing Maxima as a programming language

### DISCUSSION THREAD

**Discussion by Observatorio-de-Matematica** [community/community#73158] [Programming Help]
### Body

Hi.

How can the file extensions and the repositories with Maxima code be recognized?

The file extensions currently found in GitHub for Maxima files are .mac, .wxm and .wxmx; but the last is not readable in GitHub.
It also happens that a repo can have 90% of the files in this extensions, but a single file with an extension of a recognized language in GitHub makes that GitHub recognize the repository as 100% Matlab or something else.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by spenserblack** [comment]
You'll want to check out [linguist](https://github.com/github-linguist/linguist) for that, which is the tool that GitHub uses to identify languages in a repository, and also acts as a data source for a few other tools. If the usage is common enough, you can make a pull request to add or enhance support for the language. If it's not common enough, the next step is to check out their documentation on overrides and see if something works for your individual repository.

**Comment by Shubham18024** [comment]
oh , please add maxima, I know that their is not so much community support,
but still its use in many university across India.
